app/api/v1/endpoints/exchanges.py

```
@@ -145,34 +162,117 @@ async def _sync_server_time(self) -> bool:
+                        
+                        # Store in Redis with 1 hour expiration
+                        await self._redis.setex(self._nonce_key, 3600, str(new_nonce))
+                        
+                        self.logger.debug(
+                            "Generated Redis nonce", 
+                            nonce=new_nonce,
+                            last_nonce=last_nonce,
+                            server_time_offset=self._server_time_offset,
+                            node_id=self._node_id
+                        )
+                        return new_nonce
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)